reference/Counter/CSharp-MVUX/Counter.Windows/Package.appxmanifest

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="Counter"
+    Publisher="O=Agnes Zitte"
```

Comment:
publisher should be in name o a person or Uno organization?

Reply:
I'm not sure.
@agneszitte ?

Reply:
Adjusted